src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -1573,6 +1573,19 @@ public function setEagerLoads(array $eagerLoad)
 
         return $this;
     }
+    
+    /**
+     * Flush the given relationships being eagerly loaded.
```

Comment:
I guess this comment needs to be changed, it is the same as the method below (`withoutEagerLoads`) and is not describing intended feature.